### PR TITLE
Fix tasks complete

### PR DESCRIPTION
**Pull request by boikonataliii**
**STATUS**
READY

**DESCRIPTION**

http://prntscr.com/lhz4sn checked for completed
completed_at isn't changed http://prntscr.com/lhz7z7 after click on up/down

**CHECK LIST**
- [x] specs added
- [x] specs passed
- [ ] code coverage >90%
- [x] DB indexes added if needed
- [x] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [x] I've checked new feature as logged in and logged out user if needed
- [x] "let!" is used in specs

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -85,11 +85,12 @@
     t.datetime "updated_at", null: false
     t.integer "row_order"
     t.bigint "project_id"
+    t.boolean "section", default: false
     t.boolean "complete", default: false
-    t.boolean "section"
-    t.bigint "assignee_id"
     t.datetime "deleted_at"
     t.datetime "due_date"
+    t.bigint "assignee_id"
```
Comment (AndriySolonyna) [inline db/schema.rb:91]: I see changes in schema but don't see any migrations

File: app/controllers/account/tasks_controller.rb [hunk 2]
```diff
@@ -105,7 +105,7 @@ def remove_attachment
   def complete
     @project = parent
     @task = resource
-    @task.update(complete: true)
+    @task.update(complete: true, completed_at: Time.now)
```
Comment (AndriySolonyna) [inline app/controllers/account/tasks_controller.rb]: you don't need complete field, if a task has completed_at that means it completed

**Review by AndriySolonyna — CHANGES_REQUESTED** [review]

File: app/controllers/account/tasks_controller.rb (hunk 2 above)
Comment (igorkasyanchuk) [reply app/controllers/account/tasks_controller.rb]: completed = completed_at not nil

File: db/schema.rb (hunk 1 above)
Comment (igorkasyanchuk) [reply db/schema.rb:91]: it's fine. maybe settings of DB. same columns, just different order

File: spec/factories/users.rb [hunk 3]
```diff
@@ -73,7 +73,7 @@
         workspace.projects.each do |project|
           FactoryBot.create(:task, :future, project: project)
           FactoryBot.create(:task, :expired, project: project)
-          FactoryBot.create(:task, :completed, :expired, project: project)
+          FactoryBot.create(:task, :completed, :expired, :completed_at, project: project)
```
Comment (igorkasyanchuk) [inline spec/factories/users.rb]: rename trait to "completed"

**Review by AndriySolonyna — APPROVED** [review]

**Pull request merged**